setup.py

```diff
@@ -1,28 +1,49 @@
 #!/usr/bin/env python
-# Licensed under a 3-clause BSD style license - see LICENSE.rst
-
+from setuptools import setup  # isort:skip
 import os
+import sys
 from itertools import chain
 
-from setuptools import setup
 from setuptools.config import read_configuration
 
 ################################################################################
-# Programmatically generate some extras combos.
+# Raise helpful messages for test and build_docs commands
```

Comment:
We don't really need this in a new package? tbh could probably pull it from core now as well

Reply:
Could remove it.

Reply:
Done.